src/peer_digest.cc

```diff
@@ -349,7 +327,7 @@ peerDigestHandleReply(void *data, StoreIOBuffer receivedData)
         return;
     }
 
-    assert(fetch->pd);
+    assert(fetch->pd.raw());
```

Comment:
Remove because this function has no code dereferencing fetch->pd?

```suggestion
```

Reply:
Done.